sdks/java/core/src/main/java/org/apache/beam/sdk/util/FileIOChannelFactory.java

```diff
@@ -49,6 +50,22 @@
 public class FileIOChannelFactory implements IOChannelFactory {
   private static final Logger LOG = LoggerFactory.getLogger(FileIOChannelFactory.class);
 
+   /**
+   * Create a {@link FileIOChannelFactory} with the given {@link PipelineOptions}.
+   */
+  public static FileIOChannelFactory fromOptions(PipelineOptions options) {
+    return create();
+  }
+
+  /**
+   * Create a {@link FileIOChannelFactory}.
+   */
+  public static FileIOChannelFactory create() {
```

Comment:
Remove this method since we want people to always use fromOptions, replace existing uses with passing in an empty or null PipelineOptions. Add `@nullable` to fromOptions